src/frontends/tensorflow_common/src/op/binary_op.cpp

```diff
@@ -142,6 +142,32 @@ OutputVector translate_mul_op(const NodeContext& node) {
     set_node_name(node.get_name(), result);
     return {result};
 }
+
+OutputVector translate_addv2_op(const NodeContext& node) {
```

Comment:
please define this function here: https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/tensorflow_common/include/common_op_table.hpp#L34
replace previous translator with your one for `Add` and `AddV2` here: https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/tensorflow/src/op_table.cpp#L170